build-kubeadm-edge-snap.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+
+set -o errexit  # Exit when an individual command fails.
+set -o pipefail  # The exit status of the last command is returned.
+
+OS=${OS:-"linux"}
+ARCH=${ARCH:-"amd64"}
+
+rm -rf kubernetes
+git clone https://github.com/kubernetes/kubernetes.git
+
+cd kubernetes
+export KUBE_VERSION=`git describe --dirty`
+
+# Grab the user id and group id of this current user.
+GROUP_ID=$(id -g)
+USER_ID=$(id -u)
+
+docker run -it \
+  --rm \
+  -e "GOOS=${OS}" \
+  -e "GOARCH=${ARCH}" \
+  -v ${PWD}:/kubernetes \
```

Comment:
Might be good to quote this in case `${PWD}` has a space in it - although on jenkins, it probably won't